### DISCUSSION TITLE
Terrain texture alpha?

### DISCUSSION THREAD

**Discussion by alienatorZ** [visgl/deck.gl#7487] [Q&A] [answered]
Is there a way to set the alpha on a terrain texture?

**Comment by Pessimistress** [comment, marked as answer]
Just set `opacity` of the layer?

**Reply by alienatorZ** [reply]
Yes that works for the entire layer. I was hoping to control the opacity of the texture separately. Is there any progress towards controlling multiple layers on top of terrain?

**Reply by Pessimistress** [reply]
> I was hoping to control the opacity of the texture separately

Not sure what you mean by that. Do you have a picture to show what you are trying to achieve?

**Reply by alienatorZ** [reply]
I was trying a hacky way of putting multiple layers over terrain by creating multiple terrain layers with different textures, but deck.gl cannot handle multiple terrain layers over each other. So I was trying to alpha one layer to see the next underneath. I dont know how to create a picture to show. The issue [#7195](https://github.com/visgl/deck.gl/issues/7195) would give this functionality I think.

**Reply by Pessimistress** [reply]
You can turn off depth testing to render overlapping meshes, see https://deck.gl/docs/developer-guide/tips-and-tricks#z-fighting-and-depth-testing

**Reply by alienatorZ** [reply]
Ahh I will have to try that!
